packages/xo-server/src/xapi/isDefaultTemplate.js

```diff
@@ -0,0 +1,2 @@
+export default ({ is_default_template, other_config }) =>
+  is_default_template || other_config.default_template === 'true'
```

Comment:
You don't have to change, but just to show that destructuring is not always shorter :slightly_smiling_face: 

```suggestion
export default vmTpl => vmTpl.is_default_template || vmTpl.other_config.default_template === 'true'
```